generated/formats/bnk/compound/HircPointer.py

```diff
@@ -17,10 +17,10 @@ def __init__(self, context, arg=0, template=None, set_default=True):
 		self.template = template
 		self.io_size = 0
 		self.io_start = 0
-		self.id = HircType(self.context, 0, None)
-		self.data = SoundSfxVoice(self.context, 0, None)
-		self.data = MusicTrack(self.context, 0, None)
-		self.data = TypeOther(self.context, 0, None)
+		self.id = 0
+		self.data = 0
+		self.data = 0
+		self.data = 0
```

Comment:
If we absolutely do not care for dtype in init, can just do once per union.

Reply:
Alright, done. How do you propose handling a union where the fields have different field_debug_str ? Use the first one, print them both, or don't use them at all? At the moment all unique debug strings are printed in the file, which is probably fine, though I could maybe see it being a bit confusing.